pkg/porter/explain.go

```diff
@@ -379,3 +401,30 @@ func (p *Porter) printActionsExplainTable(bun *PrintableBundle) error {
 		}
 	return printer.PrintTable(p.Out, bun.Actions, printActionRow, "Name", "Description", "Modifies Installation", "Stateless")
 }
+
+// Dependencies
+func (p *Porter) printDependenciesExplainBlock(bun *PrintableBundle) error {
+	if len(bun.Dependencies) > 0 {
+		fmt.Fprintln(p.Out, "Dependencies:")
+		err := p.printDependenciesExplainTable(bun)
+		if err != nil {
+			return errors.Wrap(err, "unable to print dependencies table")
+		}
+	} else {
+		fmt.Fprintln(p.Out, "No dependencies defined")
+	}
+	fmt.Fprintln(p.Out, "") // force a blank line after this block
+	return nil
+}
+
+func (p *Porter) printDependenciesExplainTable(bun *PrintableBundle) error {
+	printOutputRow :=
+		func(v interface{}) []interface{} {
+			o, ok := v.(PrintableDependency)
+			if !ok {
+				return nil
+			}
+			return []interface{}{o.Alias, o.Tag}
+		}
+	return printer.PrintTable(p.Out, bun.Outputs, printOutputRow, "Alias", "Tag")
```

Comment:
Should be `bun.Dependencies`, right?

Reply:
When I test against the [wordpress bundle](https://github.com/deislabs/porter/tree/main/build/testdata/bundles/wordpress), the `mysql` dependency doesn't appear to populate correctly when using the default table format (most likely b/c of the issue mentioned above):

```
 $ porter explain
Name: wordpress
Description:
Version: 0.1.3
...
Dependencies:
Alias   Tag

```

Reply:
Sorry :(

Reply:
It looks good now when I manually tested, but, indeed, it might be a good idea to add a unit test for this... maybe similar to the ones for the `params-bundle` where it matches against expected yaml, json and table output?

Reply:
Yes, I'll make sure to have it